docs/serving/using-cert-manager-on-gcp.md

```diff
@@ -1,211 +1,250 @@
 ---
```

Comment:
@ZhiminXiang ive edited and added this file to the PR since its so closely tied to everything else. PTAL

Reply:
LGTM. I think this PR is ready to go.